.github/workflows/create-release.yml

```diff
@@ -17,6 +17,9 @@ on:
         required: false
         default: "false"
```

Comment:
Discussed with @dragonstyle off github, we can do the change. It will be better to really have those as boolean, and I don't think it will change anything on current workflow as `github.event.inputs.publish-release`. 

We would need to use `${{ inputs.pre-release }}` otherwise to get the boolean, but does not work in the composite anyway